src/winstanley/structure/WdlImplicits.scala

```diff
@@ -89,6 +89,15 @@ object WdlImplicits {
 
     def getTaskIdentifierNode: Option[ASTNode] = Option(psiElement.getNode.findChildByType(WdlTypes.TASK_IDENTIFIER_DECL))
 
+    def getRootElement: PsiElement = {
```

Comment:
ToL (ie thinking out loud - not necessary a must-fix...)

Oh, I didn't actually realize this wasn't a built-in when I saw it above.

Something like `getWdlVersionFileElement` might make it look less built-in?